Integrations/FortiGate/FortiGate.py

```diff
@@ -429,6 +467,76 @@ def create_firewall_service_request(service_name, tcp_range, udp_range):
     return response
 
 
+def ban_ip(ip_addresses_array, time_to_expire):
+    uri_suffix = 'monitor/user/banned/add_users/'
+
+    payload = {
+        'ip_addresses': ip_addresses_array
+    }
+    if time_to_expire:
+        time_to_expire_int = convert_arg_to_int(time_to_expire, "expiry")
```

Comment:
this should be handled in `ban_ip_command`.
the purpose of separating to 2 functions is: outer function will process the user input (handling arrays, numbers, dates, etc...) and inner function will handle the request logic.

here, you can use the value `0` or `None` for no expiration time